website/pages/docs/reference/destination-spec.md

```diff
@@ -66,13 +66,16 @@ Specifies the update method to use when inserting rows. The exact semantics depe
 - `overwrite`: Same as `overwrite-delete-stale`, but doesn't delete stale rows from previous `sync`s.
 - `append`: Rows are never overwritten or deleted, only appended.
 
+> Switching from `overwrite-delete-stale` or `overwrite` to `append`, or from `append` to `overwrite-delete-stale` or `overwrite` is not supported without dropping all the tables.
```

Comment:
'all' only if the user has `tables: *`, no? One can theoretically sync different tables with separate/incompatible modes as long as they are not synced in the same run

Reply:
Oh sorry got bit by automerge, but I think "all tables matched by your table config" is the most obvious interpretation of the word `all` here

Reply:
Maybe we can change it to what I said just to be clear we won't drop other tables though

Reply:
Good point, follow up in https://github.com/cloudquery/cloudquery/pull/12532